plugin/v2/plugin.go

```diff
@@ -242,11 +251,18 @@ func (p *Plugin) ComputePrivileges() types.PluginPrivileges {
 			})
 		}
 	}
-	if len(m.Capabilities) > 0 {
+	if m.Linux.DeviceCreation {
+		privileges = append(privileges, types.PluginPrivilege{
+			Name:        "device-creation",
+			Description: "",
```

Comment:
`Description` for all privileges is left empty. Can we fill it in with descriptions?